sources/packages/web/src/router/InstitutionRoutes.ts

```diff
@@ -544,6 +547,36 @@ export const institutionRoutes: Array<RouteRecordRaw> = [
           },
         ],
       },
+      {
+        path: AppRoutes.ApplicationDetail,
+        props: true,
+        components: {
+          default: ApplicationDetails,
+          sidebar: InstitutionApplicationSideBar,
+        },
+        meta: {
+          clientType: ClientIdType.Institution,
+          institutionUserTypes: [
+            InstitutionUserTypes.admin,
+            InstitutionUserTypes.user,
+          ],
+        },
+        children: [
+          {
+            path: "",
```

Comment:
I think the when even when the parent URL is entered, this will ensure that a child component is rendered. If we need to give a value here, then we will need to add logic to redirect to the child when the parent URL is requested. else we will get a blank or a partial page

Reply:
yup, I also noticed that, and it is a gap, IMO. 

`or put a value to path and assume the user is not suppose to use the parent URL as it is not complete.`, are we suppose to assume this? 😉

Reply:
_I think, when the parent URL is entered, this will ensure that a child component is rendered (a complete view/page). If we want to add a value here, then we will need to add logic to redirect to the child when the parent URL is requested. else we will get a blank or a partial page/view_


As I mentioned above there are 2 ways, 
1. one in this example
2. add redirect logic, something like `path: 'child-path', redirect: 'parent -path' ` this https://router.vuejs.org/guide/essentials/redirect-and-alias.html,

Reply:
> @ann-aot by mentioning `when parent URL is entered` I believe you meant by manually typing the URL in browser right?
> 
> If that is the case, then we have the same situtation right now in our application where if we type a parent URL instead of child in AEST/Institution student search result i.e student details parent URL, it appears to be like this.
> 
> ![image](https://user-images.githubusercontent.com/54600590/236058944-ceae1415-2f5f-4c89-8758-1634d0127299.png)
> 
> So we can either follow this way of having `path=""` for student profile route which is default for the parent or put a value to path and assume the user is not suppose to use the parent URL as it is not complete.


As I mentioned above there are 2 ways, 
1. one in this example
2. add redirect logic, something like `path: 'child-path', redirect: 'parent -path' ` this https://router.vuejs.org/guide/essentials/redirect-and-alias.html,

Reply:
> I think, when the parent URL is entered, this will ensure that a child component is rendered (a complete view/page). If we want to add a value here, then we will need to add logic to redirect to the child when the parent URL is requested. else we will get a blank or a partial page/view


As I mentioned above there are 2 ways, 
1. one in this example
2. add redirect logic, something like `path: 'child-path', redirect: 'parent -path' ` this https://router.vuejs.org/guide/essentials/redirect-and-alias.html, 


REF: https://router.vuejs.org/guide/essentials/nested-routes.html

Reply:
Yes, something we can discuss outside the PR.